src/app/components/service/services.controller.js

```diff
@@ -0,0 +1,25 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('app.service')
+    .controller('ServicesCtrl', ServicesCtrl);
+
+  ServicesCtrl.$inject = ['$scope', '$routeParams', '$q', 'gettextCatalog', 'alertService', 'Service'];
+
+  function ServicesCtrl ($scope, $routeParams, $q, gettextCatalog, alertService, Service) {
+    $scope.request = $routeParams;
+    $scope.totalItems = 0;
+    $scope.itemsPerPage = 10;
+    $scope.currentPage = 1;
+    $scope.request.limit = $scope.itemsPerPage;
+    $scope.request.offset = 0;
+    $scope.request.sort = 'name';
+
+    var queryCallback = function (clients, headers) {
```

Comment:
Could you use function names that are more descriptive? There are a lot of queryCallback functions already around the app. Maybe use something like setTotalServices here instead?